Modules/TOF/CMakeLists.txt

```diff
@@ -2,16 +2,29 @@
 
 add_library(QcTOF)
 
-target_sources(QcTOF PRIVATE src/TOFTask.cxx src/TOFCheckRawsMulti.cxx
-        src/TOFCheckRawsTime.cxx src/TOFCheckRawsToT.cxx)
+target_sources(QcTOF 
+              PRIVATE src/TOFTask.cxx
+                      src/TOFTaskCompressed.cxx
+                      src/TOFCheckDiagnostic.cxx
+                      src/TOFCheckRawsMulti.cxx
+                      src/TOFCheckRawsTime.cxx 
+                      src/TOFCheckRawsToT.cxx)
 
 target_include_directories(
   QcTOF
   PUBLIC $<INSTALL_INTERFACE:include>
          $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
-  PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/src)
-
-target_link_libraries(QcTOF PUBLIC QualityControl)
+  PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/src
+         ${O2_ROOT}/include/TOFWorkflow
+         ${O2_ROOT}/include/TOFCompression
+         ${O2_ROOT}/include/DataFormatsTOF
+         ${O2_ROOT}/include/TOFBase )
+
+target_link_libraries(QcTOF PUBLIC QualityControl
+                                    O2::TOFBase
+                                    O2::TOFWorkflow
+                                    O2::DataFormatsTOF
+                                    O2::TOFCompression)
```

Comment:
It seems that `O2::TOFCompression` is missing. Is it something that you have not yet merged into O2 ?

Reply:
Ah yes, as you say I used a branch not merged in dev. I forgot about this. I will verify with Roberto when he will open his PR. Thank you Barth!